docs/sources/mimir/manage/mimir-runbooks/_index.md

```diff
@@ -1411,6 +1411,14 @@ How to **investigate** and **fix** it:
     - Check the number of in-memory series shown on the `Mimir / Tenants` dashboard for an approximation of the number of series that will be compacted once these blocks are shipped from ingesters.
     - Check the configured `compactor_split_and_merge_shards` for the tenant. A reasonable rule of thumb is 8-10 million series per compactor shard - if the number of series per shard is above this range, increase `compactor_split_and_merge_shards` for the affected tenant(s) accordingly.
 
+##### Persistent Volume Resizing Prerequisite - jsonnet
+
+If persistent volume resizing is required for store-gateways and automatic downscaling is enabled, it must be disabled before proceeding with the resizing process. This step is necessary to prevent any unexpected downscaling by the rollout operator while updating the stateful set for each zone.
+
+Disable automatic downscaling for store-gateways by setting `store_gateway_automated_downscale_enabled` equal to `false`.
```

Comment:
was this in `$._config.store_gateway_automated_downscale_enabled` or just `$.store_gateway_automated_downscale_enabled`? maybe you can have a full jsonnet snippet here like


```
{
  _config+: {
    store_gateway_automated_downscale_enabled: flase
  }
}
```

Reply:
The first one. You are right. Let me add a snippet . Thank you

Reply:
Added here [5bbcbcc](https://github.com/grafana/mimir/pull/9939/commits/5bbcbcc7863dc866be5ac4eecfa3fcce5d8bb5be)